srcs/request_handling/RequestHandlingUtils.cpp

```
@@ -168,13 +168,13 @@ std::string getContent_type(const std::string &file_ending)
     return "application/octet-stream";
 }
 
-std::string get_pure_hostname(RequestPacket &packet)
+std::string getPureHostname(RequestPacket &packet)
```

cppcheck

[style] The function 'getPureHostname' is never used.

(unusedFunction)